common/src/main/java/ru/coderedwolf/wordlearn/common/domain/result/Product.kt

```diff
@@ -5,7 +5,7 @@ import androidx.annotation.IntDef
 /**
  * @author CodeRedWolf. Date 24.08.2019.
  */
-sealed class Result<T> {
+sealed class Product<out T> {
```

Comment:
Is it true redux naming?

Reply:
It doesn't refer to redux, it's abstraction for easier work with assync result